SA/design_concept/content/results/outlook.typ

```diff
@@ -1,28 +1,15 @@
+#import "../../../style.typ": include_section
+
 = Outlook
 Content:
 - What new questions arise?
 - What could the application look like in a complete version?
 
-== Possible IDE-like version
-// TODO: rephrase
-VisualFP should work in an IDE-style, which means that there is:
-- An editor to compose functions
-- An editor for custom types
-- A way to organize defined functions and types
-- Mean to execute functions
-
-@visualfp_ide_mockup shows a mockup of how the VisualFP IDE could look like.
-The two main elements are the sidebar, which gives the user access to pre- and self-defined functions, and the large editor-section.
-The designs proposed later in this section describe how the function-editor section of the IDE could work.
-
-Functions in the sidebar can be grouped in modules (analogous to Haskell modules), which appear like folders in the IDE.
+== Possible future enhancements
+During the development of the PoC, two possible variants of a feature complete application emerged, which are briefly described below.
 
-Regarding code execution, there are multiple considered options:
-- The defined content is loaded into an interactive GHCi session which users can interact with.
-- The user needs to define a main function. The project is then executed like any other cabal project.
-- The IDE offers a "Try it" functionality, which allows the user to execute a single function with ad-hoc parameters.
+#include_section("design_concept/content/results/outlook_ide.typ", heading_increase: 2)
 
-#figure(
-    image("../../static/visualfp_ide_mockup.png"),
-    caption: "Mockup of VisualFP IDE"
-)<visualfp_ide_mockup>
\ No newline at end of file
+// TODO: find good name for Farhad's version
+=== Farhad's vision
```

Comment:
Eventully, we should call this differently 🙃

Reply:
That's why there's a TODO ;) I just couldn't think of a good name yet.

Reply:
aah, didn't see that 😂👍